### DISCUSSION TITLE
AegisFlow: Releasing my workflow for ComfyUI/Stable Diffusion

### DISCUSSION THREAD

**Discussion by aegis72** [Comfy-Org/ComfyUI#1798] [Show and tell]
![image](https://github.com/comfyanonymous/ComfyUI/assets/118572301/98e687b3-87de-4f61-9643-f06bed126fca)

I'm releasing my two workflows for ComfyUI that I use in my job as a designer. I've worked on this the past couple of months, creating  workflows for SD XL and SD 1.5 that create project folders with automatically named and processed exports that can be used in things like photobashing, work re-interpreting, and more.

It's available here on Github at https://github.com/aegis72/aegisflow and also via gumroad at https://majorstudio.gumroad.com/l/aegisflow , I've created a user manual in PDF form as well that I think people will find useful.

**Comment by xingren23** [comment]
greet job ,thanks for you sharding your workflow.

maybe you can load AegisFlow as a web app with ComfyFlowApp , and share to others.
https://github.com/xingren23/ComfyFlowApp

**Reply by aegis72** [reply]
Let me check it out....never seen that before :)!

**Reply by rvion** [reply]
@aegis72 @xingren23 have you checked https://github.com/rvion/cushystudio too ?

Very powerful tool to make workflows great to use and simple to share :)
Lots of amazing features there + more to come (disclaimer: I made CushyStudio)

@xingren23 let's talk ? 😁

**Reply by xingren23** [reply]
hi @aegis72, do you have run your great workflow on comfyflowapp ? if so,  you could share your use case here [comfyflowapp use case](https://github.com/xingren23/ComfyFlowApp/issues/5 ), or if you have any problem with comfyflowapp, maybe i could give some help.
thanks

**Comment by xingren23** [comment]
> @aegis72 @xingren23 have you checked https://github.com/rvion/cushystudio too ?
> 
> Very powerful tool to make workflows great to use and simple to share :) Lots of amazing features there + more to come (disclaimer: I made CushyStudio)
> 
> @xingren23 let's talk ? 😁

it's very cool tools, my friends had recommend it to me yet.

yes, of course,  you could contact me xingren23 at twitter(x)

**Comment by aegis72** [comment]
Hey there! Installing it now to try it out.

[image: image.png]
